OpenRA.Game/Network/Session.cs

```diff
@@ -185,7 +185,7 @@ public class Global
 			public bool Fog = true;
 			public bool AllyBuildRadius = true;
 			public int StartingCash = 5000;
-			public string TechLevel = "none";
+			public string TechLevel = "Unrestricted";
```

Comment:
That means your reporting is wrong.  You need to be comparing against the mod-defined defaults (specified in mod.yaml), which override these global defaults.  Replacing these global placeholders with mod-specific values is the wrong approach.